ex3.js

```diff
@@ -0,0 +1 @@
+console.log( "lalalala" )
```

Comment:
a discordo do lalala, teria que ser loló
```suggestion
console.log( "loló" )
```